Dockerfile

```diff
@@ -28,7 +28,7 @@ COPY --link --from=xx / /
 RUN xx-go --wrap
 RUN --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/go/pkg/mod \
-    go build -o bin/spire-controller-manager main.go
+    go build -o bin/spire-controller-manager ./cmd/spire-controller-manager
```

Comment:
This should just be calling `make build`. I tried making that change but got this error. Its not as trivial as I thought:

```
 => ERROR [builder 4/4] RUN --mount=type=cache,target=/root/.cache/go-build     --mount=type=cache,target=/go/pkg/mod     make build                                                                                                                                                                                                       0.1s
------
 > [builder 4/4] RUN --mount=type=cache,target=/root/.cache/go-build     --mount=type=cache,target=/go/pkg/mod     make build:
#0 0.063 env: can't execute 'bash': No such file or directory
#0 0.064 env: can't execute 'bash': No such file or directory
#0 0.065 env: can't execute 'bash': No such file or directory
#0 0.065 Makefile:58: *** unsupported OS: .  Stop.
------
Dockerfile:31
--------------------
  30 |     RUN apk add --update make
  31 | >>> RUN --mount=type=cache,target=/root/.cache/go-build \
  32 | >>>     --mount=type=cache,target=/go/pkg/mod \
  33 | >>>     make build
  34 |         #go build -o bin/spire-controller-manager ./cmd/spire-controller-manager
--------------------
ERROR: failed to solve: process "/bin/sh -c make build" did not complete successfully: exit code: 2
```